tests/test_prompt.py

```diff
@@ -247,3 +256,32 @@ def test_should_read_userchoice(self, mocker, choices, context):
         )
         read_choice.assert_called_once_with('orientation', choices)
         assert expected_choice == actual_choice
+
+    def test_should_read_yes_no(self, mocker, bool_context):
+        key = 'is_alive'
+        expected_bool = False
+
+        read_bool = mocker.patch('cookiecutter.prompt.read_user_yes_no')
+        read_bool.return_value = expected_bool
+
+        config = prompt.prompt_for_config(
+            bool_context,
+            False  # Ask the user for input
+        )
+        read_bool.assert_called_once_with(key, 'True')
+        assert config[key] == expected_bool
+
+    def test_should_read_yes_no_false(self, mocker, bool_context):
+        key = 'is_alive'
+        expected_bool = True
+
+        read_bool = mocker.patch('cookiecutter.prompt.read_user_yes_no')
+        read_bool.return_value = expected_bool
+
+        bool_context['cookiecutter'][key] = False
```

Comment:
In case you are trying to test both ``True`` and ``False``, you could use a parametrized fixture.

```python
@pytest.fixture(params=[True, False])
def bool_context(request):
    return {
        'cookiecutter': {
            'is_alive': request.param
        }
    }
```